New version: Zoom.Zoom version 5.16.5.24296

bigkevuk opened the pull request:
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123526)

Commit:
New version: Zoom.Zoom version 5.16.5.24296

Commit:
Update Zoom.Zoom.installer.yaml

stephengillie commented:
Automatic Validation ended with:
> Installation failed with exit code -1978334957 
2023-10-23 13:35:41.739 [CLI ] ShellExecute installer failed: 1633 
MSI (s) (24:20) [13:35:41:337]: Product: Zoom (ARM64) -- Installation failed. 
MSI (s) (24:20) [13:35:41:337]: Windows Installer installed the product. Product Name: Zoom (ARM64). Product Version: 5.16.24296. Product Language: 1033. Manufacturer: Zoom. Installation success or error status: 1633. Installation failed with exit code -1978335226 
2023-10-23 13:35:16.640 [CLI ] ShellExecute installer failed: 1620 

It looks like the different scope/architecture combinations have different errors, which is unusual. 

- `x86` `user` 
  - installer switches appear not to be correct: 
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/044f5a91-360d-46fc-be66-69a9de970249)

- `x86` `machine` 
- This error is new and I'll need to investigate more. 
> 2023-10-23 13:35:16.640 [CLI ] ShellExecute installer failed: 1620
2023-10-23 13:35:16.640 [CLI ] Terminating context: 0x8a150006 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:1bf

- `x64` `machine` 
- This appears to be compiled for `arm64` architecture.
> MSI (s) (24:20) [13:35:41:337]: Windows Installer installed the product. Product Name: Zoom (ARM64). Product Version: 5.16.24296. Product Language: 1033. Manufacturer: Zoom. Installation success or error status: 1633.
This installation package is not supported by this processor type. Contact your product vendor.

stephengillie commented on manifests/z/Zoom/Zoom/5.16.5.24296/Zoom.Zoom.installer.yaml:
```diff
@@ -0,0 +1,102 @@
+    ProductCode: ZoomUMX
+    UpgradeCode: '{C819B794-A45C-4F27-9860-0C86492A52CC}'
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://cdn.zoom.us/prod/5.16.5.24296/arm64/ZoomInstallerFull.msi
+  InstallerSha256: 4612697667A5245D8121F0BC35C1A98198A78383F2615447B06D813C7FCF3096
+  ProductCode: '{116386E9-7E60-4E8C-BA50-4C6FCE952A90}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Zoom (ARM64)
+    DisplayVersion: 5.16.5 (24296)
+    ProductCode: '{130C5F26-7D81-4285-9F6C-C18D91503887}'
+    UpgradeCode: '{C819B794-A45C-4F27-9860-0C86492A52CC}'
+- InstallerLocale: en-US
+  Architecture: arm64
```
```suggestion
  Architecture: arm64
  InstallerType: msi
  Scope: machine
  InstallerUrl: https://cdn.zoom.us/prod/5.16.5.24296/arm64/ZoomInstallerFull.msi
  InstallerSha256: 4612697667A5245D8121F0BC35C1A98198A78383F2615447B06D813C7FCF3096
  ProductCode: '{116386E9-7E60-4E8C-BA50-4C6FCE952A90}'
  AppsAndFeaturesEntries:
  - DisplayName: Zoom (ARM64)
    DisplayVersion: 5.16.5 (24296)
    ProductCode: '{130C5F26-7D81-4285-9F6C-C18D91503887}'
    UpgradeCode: '{C819B794-A45C-4F27-9860-0C86492A52CC}'
- InstallerLocale: en-US
  Architecture: x64
```

Commit:
Update Zoom.Zoom.installer.yaml

Fixed the issue KOMAC generated file seemed to have screwed up the layout of the file. Taken a copy of the last update and updated the version numbers.

Commit:
Update Zoom.Zoom.installer.yaml

Missed another update to the version numbers in the manifest doh!

Commit:
Update Zoom.Zoom.installer.yaml

Updated the SHA256

Commit:
Update Zoom.Zoom.installer.yaml

Commit:
Update Zoom.Zoom.installer.yaml

Another attempt to get the SHA256 hash right
